doc/classes/Node.xml

```diff
@@ -885,6 +897,13 @@
 				Set to [code]true[/code] to allow all nodes owned by [param node] to be available, and editable, in the Scene dock, even if their [member owner] is not the scene root. This method is intended to be used in editor plugins and tools, but it also works in release builds. See also [method is_editable_instance].
 			</description>
 		</method>
+		<method name="set_exposed_in_scene">
+			<return type="void" />
+			<param index="0" name="enable" type="bool" />
+			<description>
+				Sets this node's state as an exposed node in this [PackedScene]. This allows the node to be accessed in the editor when the scene is instantiated in another.
```

Comment:
This PackedScene?
Also not sure what is the difference from the other method. `owner` usually refers to scene root, which would be equivalent of "this scene".

Reply:
Will supe up these descriptions!

for reference here's the general idea:
exposed_in_owner  // The node has been exposed within the base scene of it's owner
exposed_in_scene   // The node has been re-exposed within an instantiated scene
exposed                  // The node has been exposed and is visible in the editor